paper-server/src/main/java/org/bukkit/craftbukkit/inventory/view/builder/CraftAbstractInventoryViewBuilder.java

```diff
@@ -0,0 +1,56 @@
+package org.bukkit.craftbukkit.inventory.view.builder;
+
+import com.google.common.base.Preconditions;
+import io.papermc.paper.adventure.PaperAdventure;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.world.inventory.AbstractContainerMenu;
+import net.minecraft.world.inventory.MenuType;
+import org.bukkit.craftbukkit.entity.CraftHumanEntity;
+import org.bukkit.craftbukkit.util.CraftChatMessage;
+import org.bukkit.entity.HumanEntity;
+import org.bukkit.inventory.InventoryView;
+import org.bukkit.inventory.view.builder.InventoryViewBuilder;
+import org.jetbrains.annotations.NotNull;
+
+public abstract class CraftAbstractInventoryViewBuilder<V extends InventoryView> implements InventoryViewBuilder<V> {
```

Comment:
rename to CraftInventoryViewBuilder

Reply:
No its an abstract class, Naming conventions for abstract classes should include that they are abstract

Reply:
> No its an abstract class, Naming conventions for abstract classes should include that they are abstract. This creates more clarity that it is not a standalone and is dependent on some further implementation

There is no such convention. This creates a redundancy since you use the abstract keyword anyway and it's not even API so it's even less important to follow some convention you set for yourself.
Also you make the point of "This creates more clarity that it's not a standalone". This makes no sense as the presence of the `abstract` keyword already says as much. You can't instantiate an abstract class. Therefore you cannot really use an abstract class. An abstract class never is a standalone.

Reply:
Whether or not you like the naming stripping the word `Abstract` from this class name furthers itself from a standard of other internal classes, feel free to look yourself to confirm, but also doesn't actually improve the content of the PR. Not worth continuing a debate over such a minutely unimportant detail that follows an arguably useful common distinction already made within the internals.

Reply:
Not wanna fuel the fire here but even if it is a "standard" in the current state of the project it doesn't have to be continued
Just saying, when/if this gets merged, it is worth adhering to proper standards.

Reply:
`Abstract` seems fine to me. I followed a similar convention all over the lifecycle event system implementation.